interp/notation.ml

```diff
@@ -707,11 +707,10 @@ end
 
 (** Conversion from bigint to int63 *)
 let rec int63_of_pos_bigint i =
-  let open Bigint in
-  if equal i zero then Uint63.of_int 0
+  if Z.(equal i zero) then Uint63.of_int 0
   else
-    let (quo,rem) = div2_with_rest i in
-    if rem then Uint63.add (Uint63.of_int 1)
+    let quo, remi = Z.div_rem i (Z.of_int 2) in
+    if Z.(equal remi zero) then Uint63.add (Uint63.of_int 1)
```

Comment:
The logic here is flipped.  The true case is for `i` odd

Reply:
Fixed, thanks @JasonGross ; note what with the new Z api many things can be implemented better.